packages/ui/src/theme/styles/overrides/tokens.css

```
@@ -35,30 +35,43 @@
-    /**
-   * Optional sans fallback — exposed for cases that explicitly want
-   * proportional Geist (e.g. very long prose passages).
-   */
-    --cp-font-family-sans: 'Geist', ui-sans-serif, system-ui, sans-serif;
-    /**
-   * Decorative pixel-art font — opt-in for brand marks, headlines, etc.
-   * Apply with `font-family: var(--cp-font-family-pixel)` on the target.
-   */
-    --cp-font-family-pixel: 'Geist Pixel Square', ui-sans-serif, system-ui, sans-serif;
+    --cp-ff-sans:
+      'Inter var experimental', 'Inter var', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)